samples/metaapp/src/main/java/com/sourcepointmeta/metaapp/tv/properties/PropertyListFragmentTv.kt

```diff
@@ -53,6 +52,19 @@ class PropertyListFragmentTv : VerticalGridSupportFragment(), OnItemViewClickedL
         requireContext().showPropertyDetail(propDto.propertyName)
     }
 
+    private fun titleInit(){
```

Comment:
this method can be turned in an extension

Reply:
@carmelo-iriti done